pages/linux/pacman-upgrade.md

```diff
@@ -6,24 +6,24 @@
 
 - Install one or more packages from files:
 
-`sudo pacman --upgrade {{path/to/package1.pkg.tar.zst}} {{path/to/package2.pkg.tar.zst}}`
+`sudo pacman -U {{path/to/package1.pkg.tar.zst}} {{path/to/package2.pkg.tar.zst}}`
 
 - Install a package without prompting:
 
-`sudo pacman --upgrade --noconfirm {{path/to/package.pkg.tar.zst}}`
+`sudo pacman -U --noconfirm {{path/to/package.pkg.tar.zst}}`
 
 - Overwrite conflicting files during a package installation:
 
-`sudo pacman --upgrade --overwrite {{path/to/file}} {{path/to/package.pkg.tar.zst}}`
+`sudo pacman -U --overwrite {{path/to/file}} {{path/to/package.pkg.tar.zst}}`
 
-- Install a package, skipping the dependency version checks:
+- Install a package, skipping the dependency [(d)] version checks:
 
-`sudo pacman --upgrade --nodeps {{path/to/package.pkg.tar.zst}}`
+`sudo pacman -Ud {{path/to/package.pkg.tar.zst}}`
 
-- List packages that would be affected (does not install any packages):
+- List packages ([p]) that would be affected by upgrade (does not install any packages):
```

Comment:
```suggestion
- [p]rint packages that would be affected by upgrade (does not install any packages):
```
I wasn't able to come up with a word order that made grammatical sense. This will have to do unless you come up with a better one.

Reply:
```suggestion
- Fetch and [p]rint packages that would be affected by upgrade (does not install any packages):
```

Yeah, I agree. The main concern is having it lower-case.

Reply:
That looks food. The grammatical case isn't that big of a deal. I've seen a couple of pages start with a "[p]rint".